ocm.py

```diff
@@ -0,0 +1,44 @@
+#!/bin/python
+import requests
+
+def call_ocm(endpoint, token, parameters=None):
+    requests.packages.urllib3.disable_warnings()
+    r = requests.get("{}/{}".format("https://api.openshift.com/api/", endpoint),
+            params=parameters, 
+            headers = {
+                'Accept': 'application/json',
+                'Authorization': 'Bearer {}'.format(token),
+                'User-Agent': 'pythonCall'
```

Comment:
let's be nice and add [a version][1].  Module-level global:

```python
__version__ = '0.1.0'
```
and here:
```python
'User-Agent': 'cluster-support-bot/{}'.format(__version__),
```
or some such?

[1]: https://tools.ietf.org/html/rfc2616#section-3.8

Reply:
Following RFC, sounds like a good idea :+1: